terraform/modules/cloudwatch/alarms.tf

```diff
@@ -0,0 +1,98 @@
+module "aws-rds-alarms" {
+  source                               = "lorenzoaiello/rds-alarms/aws"
+  version                              = "2.2.0"
```

Comment:
Should this be configured somewhere?

Reply:
terraform init installs the module and it doesn't need any other configuration

Reply:
Ah sorry, I meant the version, should this be in vars, or is it okay just inline? How often might it be changed?

Reply:
No prob, from the examples I've seen they all use the version number inline but will look it up

Reply:
Yeah having the version hard-coded seems to be the done thing. Also looking at the releases for each library, they don't seem to update very often. The Lorenzoaiello one has only had two releases in total since it started